README.md

```diff
@@ -15,4 +17,58 @@ operating system.
 
 #### `ExitCode`
 
-An enum-type class that contains known exit codes.
\ No newline at end of file
+An enum-type class that contains known exit codes.
+
+#### `ProcessManager`
+
+A higher-level service for spawning and communicating with processes.
+
+##### Use `spawn` to create a process similar to how a bash/shell script would:
```

Comment:
Instead of `similar to how a bash/shell script would` I'd just say what it does – forwards stdout/err – and...what else?